Documentation/ceph-upgrade.md

```diff
@@ -471,6 +463,18 @@ It's best to run `ceph config-key dump` again to verify references to
 
 See for more information, see here: https://github.com/rook/rook/issues/9185
 
+### **Rename CephBlockPool device_health_metrics pool when upgrading to Quincy v17**
+In Ceph Quincy (v17), the `device_health_metrics` pool was renamed to `.mgr`. Ceph will perform this
+migration automatically. If you do not use CephBlockPool to customize the configuration of the
+`device_health_metrics` pool, you don't need to do anything further here.
+
+If you do use CephBlockPool to customize the configuration of the `device_health_metrics` pool, you
+should create a new CephBlockPool to configure the `.mgr` built-in pool when starting the upgrade.
+You can reference the example
+[builtin mgr pool](https://github.com/rook/rook/blob/{{ branchName }}/deploy/examples/pool-builtin-mgr.yaml).
+Once the upgrade is complete, it is safe to delete the old CephBlockPool that represents the
+`device_health_metrics` pool.
+
```

Comment:
I did some pretty extensive testing here to see what happens...
- if the pool is renamed versus created new
- when/if the `device_health_metrics` one is deleted
- what happens when the operator restarts

... and I believe this is the single workflow that will work for all cases without the possibility of leaving  a stray `device_health_metrics` pool and without causing the `device_health_metrics` pool to be destroyed prematurely.

Reply:
Although... maybe it's best to not create it until after in case content present in device_health_metrics wouldn't be copied/transferred properly?